app/src/main/java/com/garethevans/church/opensongtablet/ProcessSong.java

```diff
@@ -1193,6 +1193,8 @@ private TableRow capolinetoTableRow(Context c, Preferences preferences, int lyri
             capobit.setTextSize(fontsize * preferences.getMyPreferenceFloat(c,"scaleChords",1.0f));
             capobit.setTextColor(lyricsCapoColor);
             capobit.setTypeface(StaticVariables.typefaceChords);
+            // IV - Choice to make bold
+            capobit.setPaintFlags(capobit.getPaintFlags()| Paint.FAKE_BOLD_TEXT_FLAG);
```

Comment:
Need to check where the option to switch this on and off is (same for headings)

Reply:
There are no options yet.  This change presents songs more like a typical song print, where chords and section titles are bold.  This and use of Roboto font (much better width to the space character) gives an on screen display close to a typical printed page look (probably a users initial expectation).

Yes these should be settings - particularly as current users might not like!  Did not want to fiddle with menus!

Reply:
Added a toggle (in fonts settings page) to switch this on.  Added to IanTest branch and will push to master after completing pull request.